editoast/src/schema/rolling_stock/mod.rs

```diff
@@ -27,6 +27,7 @@ pub struct RollingStock {
     pub loading_gauge: String,
     pub metadata: RollingStockMetadata,
     pub power_restrictions: Option<JsonValue>,
+    pub energy_sources: Vec<EnergySource>,
```

Comment:
This field could be optional (default: empty list). This allows to remain compatible with the old schema.
You will no longer need to specify `"energy_sources": []` for all examples of rolling stock.

```suggestion
    #[serde(default)]
    pub energy_sources: Vec<EnergySource>,
```